init-fcv.js

```diff
@@ -0,0 +1,2 @@
+db = db.getSiblingDB("admin");
+db.adminCommand({ setFeatureCompatibilityVersion: "7.0", confirm: true });

```

Comment:
This doc https://www.mongodb.com/docs/manual/release-notes/8.0/#downgrade-policy states `Binary downgrades aren't supported for MongoDB Community Edition.`. Can you make sure that your local db is shutdown and then start it up with this in place and check that the active compatibility version is actually 7.0 (using the get command). The docs seem to tell me this doesn't work on the community edition (which I thought the docker container used). I think you'll want to do something like `docker compose down` so that it fully stops your existing deployment and then kick it back up to see.